### PR TITLE
[Cart] Update cart to v3

### PR DESCRIPTION
**Pull request by tanhaok**

### CODE REVIEW

File: cart/src/main/resources/application.properties [hunk 1]
```diff
@@ -2,16 +2,18 @@ server.port=8084
...
+logging.pattern.level=%5p [${spring.application.name:},%X{traceId:-},%X{spanId:-}]
 
 spring.security.oauth2.resourceserver.jwt.issuer-uri=http://identity/auth/realms/Yas
 
 yas.services.media=http://api.yas.local/media
 yas.services.product=http://api.yas.local/product
 
 spring.datasource.driver-class-name=org.postgresql.Driver
-spring.datasource.url=jdbc:postgresql://localhost:30007/cart
+spring.datasource.url=jdbc:postgresql://localhost:5432/cart
```
Comment (PhaciusPhat) [inline cart/src/main/resources/application.properties:16]: fix this port for the docker

Comment (PhaciusPhat) [reply cart/src/main/resources/application.properties:16]: When we build docker images, our database will run in port 30007, not 5432.

Comment (tanhaok) [reply cart/src/main/resources/application.properties:16]: Please help to read README.MD again. In local env for test purpose Postgres also listen on port 30007. I think you should use docker-compose b/c Dockerfile no longer updated.

Comment (PhaciusPhat) [reply cart/src/main/resources/application.properties:16]: Oh, sorry about my mistake.

**Comment by thiennn** [comment]
We will need to override the to be able to override the zipkin endpoint.
Consider to move the url to application setting

        @Bean
	public ZipkinSpanExporter zipkinSpanExporter() {
		return ZipkinSpanExporter.builder()
						.setEndpoint("http://tempo:9411/api/v2/spans")
						.build();
	}

**Pull request merged**